paymentsheet/src/test/java/com/stripe/android/model/PaymentIntentFixtures.kt

```diff
@@ -381,9 +381,9 @@ internal object PaymentIntentFixtures {
                 "card"
             ],
             "payment_method_options": {
-               "card": {
-                "setup_future_usage": null
-               }
+                 "card": {
+                   "setup_future_usage": "on_session"
```

Comment:
Previously we were checking if the key exists in the map, I am not sure if that is the correct logic. Shouldn't we be checking if the key has a value?